.prettierrc

```
@@ -1,7 +1,20 @@
 {
-  "trailingComma": "es5",
-  "tabWidth": 4,
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "embeddedLanguageFormatting": "auto",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxSingleQuote": false,
+  "printWidth": 80,
+  "proseWrap": "preserve",
+  "quoteProps": "as-needed",
+  "requirePragma": false,
   "semi": true,
+  "singleAttributePerLine": false,
   "singleQuote": true,
-  "printWidth": 130
+  "tabWidth": 2,
```

```suggestion
  "tabWidth": 4,
```

Can you please change this to tab width 4, reformat for the entire codebase, and then push your changes again? I prefer width 4 for visibility. I am unable to do a thorough review at this moment, but please do expect one some time soon; so sorry for the delay, but I promise I will get back to you both with my thoughts! Thanks!